#goal definition
geometry_msgs/PointStamped central_point
float64 height
float64 width
float64 step_length
std_msgs/Duration duration_per_point
---
#result definition
bool success
time action_end
---
#feedback definition
time action_start
